CONTRIBUTING.md

```diff
@@ -6,6 +6,10 @@
 npm install
 ```
 
+### Side Note about Angular 10
+You will need to have a very recent version of node installed (tested with 14.5.0) successful build lib and portal. 
```

Comment:
I am using Node 12, what was the issue you got, can you share?

Reply:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/1403225/86485761-d9adb380-bd59-11ea-9291-ce3b1cb2a59d.png">

Maybe I did not realized some issues with v13. Works with 12 as well.